doc/rst/usingchapel/prereqs.rst

```diff
@@ -112,7 +112,12 @@ We have used the following commands to install the above prerequisites:
       make
       sudo make install
       sudo update-alternatives --install /usr/bin/cmake cmake /usr/local/bin/cmake 1
-      sudo yum install llvm-devel clang clang-devel
+      sudo yum install gcc10 gcc10-c++
+      sudo export CC=gcc10-gcc
+      sudo export CXX=gcc10-g++
+      sudo export CHPL_HOST_CC=gcc10-gcc
+      sudo export CHPL_HOST_CXX=gcc10-g++
+      sudo export CHPL_LLVM=bundled
```

Comment:
sudo is not required for these

Reply:
nit, could you make a another script file called "set-llvm-bundled" or something so that it gets auto-generated? Not sure its that necessary as its just one manual change